cmake/modules/boards.cmake

```diff
@@ -62,6 +62,19 @@ if(NOT unittest IN_LIST Zephyr_FIND_COMPONENTS)
 endif()
 
 string(FIND "${BOARD}" "@" REVISION_SEPARATOR_INDEX)
+string(FIND "${BOARD}" "/" IDENTIFIER_SEPARATOR_INDEX)
+
+if(NOT (REVISION_SEPARATOR_INDEX EQUAL -1 OR IDENTIFIER_SEPARATOR_INDEX EQUAL -1))
+  if(REVISION_SEPARATOR_INDEX GREATER IDENTIFIER_SEPARATOR_INDEX)
+    message(FATAL_ERROR "Invalid revision / variant format, format is: <board>@<revision>/<variant>")
```

Comment:
Should variant be changed to identifier?

Reply:
it sure should.
Thanks for catching

Reply:
@tejlmand I think this is still relevant

Reply:
fixed